python/pyspark/testing/utils.py

```diff
@@ -127,3 +128,18 @@ def search_jar(project_relative_path, sbt_jar_name_prefix, mvn_jar_name_prefix):
         raise Exception("Found multiple JARs: %s; please remove all but one" % (", ".join(jars)))
     else:
         return jars[0]
+
+
+class ExecThread(threading.Thread):
```

Comment:
I think we can just have it within the test. Let's don't add it in the util for now.

Reply:
in the module of test_worker or as an inner class of `WorkerTests`?

Reply:
Can we just remove this class?